test/runtests.jl

```diff
@@ -6,11 +6,12 @@ using Test
     
     @testset "Low-Level-API" begin
```

Comment:
The name of the test and also the file `low-level-API` is a bit confusing. What we are actually testing here is whether the bindings have been correctly built right? If so, then lets just call this `Build` or `test_build.`